packages/core/test/insured-bridge/BridgeAdmin.js

```diff
@@ -32,6 +32,7 @@ const defaultGasLimit = 1_000_000;
 const defaultGasPrice = toWei("1", "gwei");
 const defaultL1CallValue = 10_000_000_000;
 const maxSubmissionCost = 10_000_000_000;
+const l2GasData = [defaultL1CallValue, defaultGasLimit, defaultGasPrice, maxSubmissionCost];
```

Comment:
this same chink of call data is fed into every function. I made a variable that I then spread within the tests. this drastically decreased the number of lines in this test file.